docs/setup/settings.asciidoc

```diff
@@ -147,6 +147,36 @@ will default to `true`.
 `logging.quiet:`:: *Default: false* Set the value of this setting to `true` to
 suppress all logging output other than error messages.
 
+`logging.rotate:`:: Specifies the options for the logging rotate feature.
+When not defined, all the sub options defaults would be applied.
+The following example shows a valid logging rotate configuration:
++
+--
+    logging.rotate:
+      enabled: true
+      everyBytes: 10485760
+      keepFiles: 10
+--
+
+`logging.rotate.enabled:`:: *Default: false* Set the value of this setting to `true` to
+enable log rotation. If you do not have a `logging.dest` set that is different from `stdout`
+that feature would not take any effect.
+
+`logging.rotate.everyBytes:`:: *Default: 10485760* The maximum size of a log file (that is `not an exact` limit). After the 
+limit is reached, a new log file is generated. The default size limit is 10485760 (10 MB) and
+this option should be at least greater than 1024.
+
+`logging.rotate.keepFiles:`:: *Default: 7* The number of most recent rotated log files to keep 
```

Comment:
should the name be clear that is `number`? `numberOfFiles`-ish?

Reply:
I've followed the same api Beats are using in order to try to not diverse too much. I think it won't be a big problem as the documentation clearly refers it is a number and we also have `JOI`.